### PR TITLE
Add Indonesian Translation for Security Overview

### PR DESCRIPTION
**Pull request by phanama**
Addresses Security Overview page for #15341 

/assign @danninov

### CODE REVIEW

File: content/id/docs/concepts/security/overview.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+title: Ikhtisar Keamanan _Cloud Native_
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Keamanan Kubernetes (dan keamanan secara umum) adalah sebuah topik sangat besar yang memiliki banyak bagian yang sangat berkaitan satu sama lain. Pada masa sekarang ini di mana perangkat lunak _open source_ telah diintegrasi ke dalam banyak sistem yang membantu berjalannya aplikasi web, ada beberapa konsep menyeluruh yang dapat membantu intuisimu untuk berpikir tentang keamanan secara menyeluruh. Panduan ini akan mendefinisikan sebuah cara/model berpikir untuk beberapa konsep umum mengenai Keamanan _Cloud Native_. Cara berpikir ini sepenuhnya subjektif dan kamu sebaiknya hanya menggunakannya apabila ini membantumu berpikir tentang di mana harus mengamankan _stack_ perangkat lunakmu.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Keamanan Kubernetes (dan keamanan secara umum) adalah sebuah topik sangat luas yang memiliki banyak bagian yang sangat berkaitan satu sama lain. Pada masa sekarang ini di mana perangkat lunak _open source_ telah diintegrasi ke dalam banyak sistem yang membantu berjalannya aplikasi web, ada beberapa konsep menyeluruh yang dapat membantu intuisimu untuk berpikir tentang konsep keamanan secara menyeluruh. Panduan ini akan mendefinisikan sebuah cara/model berpikir untuk beberapa konsep umum mengenai Keamanan _Cloud Native_. Cara berpikir ini sepenuhnya subjektif dan kamu sebaiknya hanya menggunakannya apabila ini membantumu berpikir tentang di mana harus mengamankan _stack_ perangkat lunakmu.
```

File: content/id/docs/concepts/security/overview.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+## 4C pada Keamanan _Cloud Native_
+
+Mari memulainya dengan sebuah diagram yang dapat membantumu mengerti bagaimana berpikir tentang keamanan dalam bentuk beberapa lapisan.
+{{< note >}}
+Pendekatan berlapis ini memperkuat pendekatan [_defense in depth_](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) terhadap keamanan, yang secara luas dianggap sebagai praktik terbaik untuk mengamankan sistem-sistem perangkat lunak. 4C tersebut adalah Cloud, Cluster, Container, dan Code.
+{{< /note >}}
+
+{{< figure src="/images/docs/4c.png" title="The 4C's of Cloud Native Security" >}}
+
+Seperti yang dapat kamu lihat dari gambar di atas, setiap dari 4C tersebut bergantung pada keamanan dari kotak yang lebih besar di mana mereka berada. Hampir tidak mungkin untuk mengamankan sistem terhadap standar-standar keamanan yang buruk pada Cloud, Container, dan Code hanya dengan menangani keamanan pada level kode. Akan tetapi, apabila semua area tersebut ditangani dengan baik, maka menambahkan keamanan ke dalam kode kamu akan memperkuat landasan yang sudah kuat. Area-area yang menjadi perhatian ini akan dideskripsikan lebih mendalam di bawah.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Seperti yang dapat kamu lihat dari gambar di atas, setiap dari 4C tersebut bergantung pada keamanan dari kotak yang lebih besar di mana mereka berada. Hampir tidak mungkin untuk mengamankan sistem terhadap standar-standar keamanan yang buruk pada Cloud, Container, dan Code hanya dengan menangani keamanan pada lapisan kode. Akan tetapi, apabila semua area tersebut ditangani dengan baik, maka menambahkan keamanan ke dalam kode kamu akan memperkuat landasan yang sudah kuat. Area-area yang menjadi perhatian ini akan dideskripsikan lebih mendalam di bawah.
```

File: content/id/docs/concepts/security/overview.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+IaaS Provider        | Link |
+-------------------- | ------------ |
+Alibaba Cloud | https://www.alibabacloud.com/trust-center |
+Amazon Web Services | https://aws.amazon.com/security/ |
+Google Cloud Platform | https://cloud.google.com/security/ |
+IBM Cloud | https://www.ibm.com/cloud/security |
+Microsoft Azure | https://docs.microsoft.com/en-us/azure/security/azure-security |
+VMWare VSphere | https://www.vmware.com/security/hardening-guides.html |
+
+Jika kamu mengoperasikan perangkat keras kamu sendiri, atau penyedia layanan cloud yang berbeda, kamu perlu merujuk pada dokumentasimu untuk praktik keamanan terbaik.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Jika kamu mengoperasikan perangkat keras kamu sendiri, atau penyedia layanan cloud yang berbeda, kamu perlu merujuk pada dokumentasi penyedia layanan cloud yang kamu pakai untuk praktik keamanan terbaik.
```

File: content/id/docs/concepts/security/overview.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+Akses terhadap etcd | Akses terhadap etcd (tempat penyimpanan data Kubernetes) harus dibatasi hanya untuk Master-master saja. Bergantung pada konfigurasimu, kamu sebaiknya juga mengusahakan koneksi etcd menggunakan TLS. Informasi lebih lanjut dapat ditemukan di sini: https://github.com/etcd-io/etcd/tree/master/Documentation#security |
+Enkripsi etcd | Di mana pun kita dapat melakukannya, mengenkripsi semua data saat diam (_at rest_) pada semua _drive_, dan sejak etcd menyimpan keadaan seluruh kluster (termasuk Secret-secret), _disk_-nya sebaiknya kita enkripsi saat diam. |
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Enkripsi etcd | Di mana pun kita dapat melakukannya, mengenkripsi semua data saat diam (_at rest_) pada semua _drive_, dan sejak etcd menyimpan keadaan seluruh klaster (termasuk Secret-secret), _disk_-nya sebaiknya kita enkripsi saat diam. |
```

File: content/id/docs/concepts/security/overview.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+Microsoft Azure | https://docs.microsoft.com/en-us/azure/security/azure-security |
+VMWare VSphere | https://www.vmware.com/security/hardening-guides.html |
+
+Jika kamu mengoperasikan perangkat keras kamu sendiri, atau penyedia layanan cloud yang berbeda, kamu perlu merujuk pada dokumentasimu untuk praktik keamanan terbaik.
+
+### Tabel Panduan Umum Infrastruktur
+
+Area yang Menjadi Perhatian untuk Infrastruktur Kubernetes | Rekomendasi |
+--------------------------------------------- | ------------ |
+Akses Jaringan terhadap API Server (Master-master) | Secara Ideal, semua akses terhadap Master-master Kubernetes tidak diizinkan secara publik pada internet, dan dikontrol oleh daftar kendali akses (_network ACL_) yang dibatasi untuk kumpulan alamat IP yang dibutuhkan untuk mengelola kluster. |
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Akses Jaringan terhadap API Server (Master-master) | Secara Ideal, semua akses terhadap Master-master Kubernetes tidak diizinkan secara publik pada internet, dan dikontrol oleh daftar kendali akses (_network ACL_) yang dibatasi untuk kumpulan alamat IP yang dibutuhkan untuk mengelola klaster. |
```

File: content/id/docs/concepts/security/overview.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+Akses Jaringan terhadap Node-node (Server-server Worker)| Node-node harus dikonfigurasikan untuk _hanya_ menerima koneksi-koneksi (melalui daftar kendali akses) dari Master-master pada porta-porta (_port_) yang telah ditentukan, dan menerima koneksi-koneksi dari Service-service Kubernetes dengan tipe NodePort dan LoadBalancer. Apabila memungkinkan, Node-node tersebut sebaiknya tidak diekspos pada internet publik sama sekali. |
+Akses Kubernetes terhadap API Penyedia Layanan Cloud | Setiap penyedia layanan cloud perlu memberikan kumpulan izin yang berbeda-beda untuk Master-master dan Node-node Kubernetes, sehingga rekomendasi ini sifatnya lebih umum. Praktik terbaiknya adalah untuk memberikan kluster akses terhadap penyedia layanan cloud yang mengikuti [_principle of least privilege_ (prinsip hak istimewa paling sedikit)](https://en.wikipedia.org/wiki/Principle_of_least_privilege) untuk sumber daya yang kluster tersebut perlukan untuk dikelola. Sebuah contoh untuk Kops di AWS dapat ditemukan di sini: https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles |
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Akses Kubernetes terhadap API Penyedia Layanan Cloud | Setiap penyedia layanan cloud perlu memberikan kumpulan izin yang berbeda-beda untuk Master-master dan Node-node Kubernetes, sehingga rekomendasi ini sifatnya lebih umum. Praktik terbaiknya adalah untuk memberikan klaster akses terhadap penyedia layanan cloud yang mengikuti [_principle of least privilege_ (prinsip hak istimewa paling sedikit)](https://en.wikipedia.org/wiki/Principle_of_least_privilege) untuk sumber daya yang kluster tersebut perlukan untuk dikelola. Sebuah contoh untuk Kops di AWS dapat ditemukan di sini: https://github.com/kubernetes/kops/blob/master/docs/iam_roles.md#iam-roles |
```

File: content/id/docs/concepts/security/_index.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
+title: "Security"
```
Comment (irvifa) [inline content/id/docs/concepts/security/_index.md]:
```suggestion
title: "Keamanan"
```

File: content/id/docs/concepts/security/overview.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+## Cloud
+
+Dalam banyak hal, Cloud (atau server-server _co-located_, atau pusat data/_data center_ korporat) adalah [_trusted computing base_ (basis komputasi yang dipercaya)](https://en.wikipedia.org/wiki/Trusted_computing_base) dari sebuah kluster Kubernetes. Jika komponen-komponen tersebut rentan secara keamanan (atau dikonfigurasi dengan cara yang rentan), maka sesungguhnya tidak ada cara untuk menjamin keamanan dari komponen-komponen apa pun yang dibangun di atas basis komputasi ini. Memberikan rekomendasi untuk keamanan cloud berada di luar lingkup panduan ini, karena setiap penyedia layanan cloud dan beban kerja pada dasarnya berbeda-beda. Berikut beberapa tautan menuju beberapa dokumentasi penyedia layanan cloud yang populer untuk keamanan maupun untuk memberikan panduan umum untuk mengamankan infrastruktur yang menjadi basis sebuah kluster Kubernetes.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Dalam banyak hal, Cloud (atau server-server _co-located_, atau pusat data/_data center_ korporat) adalah [_trusted computing base_ (basis komputasi yang dipercaya)](https://en.wikipedia.org/wiki/Trusted_computing_base) dari sebuah klaster Kubernetes. Jika komponen-komponen tersebut rentan secara keamanan (atau dikonfigurasi dengan cara yang rentan), maka sesungguhnya tidak ada cara untuk menjamin keamanan dari komponen-komponen apa pun yang dibangun di atas basis komputasi ini. Memberikan rekomendasi untuk keamanan cloud berada di luar lingkup panduan ini, karena setiap penyedia layanan cloud dan beban kerja pada dasarnya berbeda-beda. Berikut beberapa tautan menuju beberapa dokumentasi penyedia layanan cloud yang populer untuk keamanan maupun untuk memberikan panduan umum untuk mengamankan infrastruktur yang menjadi basis sebuah klaster Kubernetes.
```

File: content/id/docs/concepts/security/overview.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+Akses terhadap etcd | Akses terhadap etcd (tempat penyimpanan data Kubernetes) harus dibatasi hanya untuk Master-master saja. Bergantung pada konfigurasimu, kamu sebaiknya juga mengusahakan koneksi etcd menggunakan TLS. Informasi lebih lanjut dapat ditemukan di sini: https://github.com/etcd-io/etcd/tree/master/Documentation#security |
+Enkripsi etcd | Di mana pun kita dapat melakukannya, mengenkripsi semua data saat diam (_at rest_) pada semua _drive_, dan sejak etcd menyimpan keadaan seluruh kluster (termasuk Secret-secret), _disk_-nya sebaiknya kita enkripsi saat diam. |
+
+## Cluster
+
+Bagian ini akan memberikan tautan-tautan untuk mengamankan beban-beban kerja di dalam Kubernetes. Ada dua area yang menjadi perhatian untuk mengamankan Kubernetes:
+
+* Mengamankan komponen-komponen yang dapat dikonfigurasi yang membentuk kluster
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
* Mengamankan komponen-komponen yang dapat dikonfigurasi yang membentuk klaster
```

File: content/id/docs/concepts/security/overview.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+* Mengamankan komponen-komponen yang berjalan di dalam kluster
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
* Mengamankan komponen-komponen yang dijalankan di dalam klaster
```

File: content/id/docs/concepts/security/overview.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+
+### Komponen-komponen _dari_ Cluster
+
+Jika kamu mau menjaga klustermu dari akses yang tidak disengaja atau yang bersifat serangan, dan mengadopsi praktik yang baik, baca dan ikutilah nasihat untuk [mengamankan klustermu](/docs/tasks/administer-cluster/securing-a-cluster/).
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Jika kamu mau menjaga klustermu dari akses yang tidak disengaja atau yang bersifat serangan, dan mengadopsi praktik yang baik, baca dan ikutilah nasihat untuk [mengamankan klastermu](/docs/tasks/administer-cluster/securing-a-cluster/).
```

File: content/id/docs/concepts/security/overview.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+### Komponen-komponen _di dalam_ Cluster (aplikasimu)
+
+Bergantung pada permukaan yang dapat diserang dari aplikasimu, kamu mungkin mau berfokus pada aspek keamanan yang spesifik. Sebagai contoh, jika kamu menjalankan sebuah layanan (kita sebut Layanan A) yang kritikal di dalam rantai sumber daya lainnya dan sebuah beban kerja terpisah (kita sebut Layanan B) yang rentan terhadap serangan _resource exhaustion_, dengan tidak menyetel limit untuk sumber daya maka kamu juga menaruh risiko terhadap Layanan A. Berikut tabel tautan-tautan menuju hal-hal yang perlu diperhatikan untuk mengamankan beban-beban kerja yang berjalan di dalam Kubernetes.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Bergantung pada permukaan yang dapat diserang dari aplikasimu, kamu mungkin ingin berfokus pada aspek keamanan yang spesifik. Sebagai contoh, jika kamu menjalankan sebuah layanan (kita sebut Layanan A) yang kritikal di dalam rantai sumber daya lainnya dan sebuah beban kerja terpisah (kita sebut Layanan B) yang rentan terhadap serangan _resource exhaustion_, dengan tidak menyetel limit untuk sumber daya maka kamu juga menaruh risiko terhadap Layanan A. Berikut tabel tautan-tautan menuju hal-hal yang perlu diperhatikan untuk mengamankan beban-beban kerja yang berjalan di dalam Kubernetes.
```

File: content/id/docs/concepts/security/overview.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+
+Area yang Menjadi Perhatian untuk Keamanan Beban Kerja | Rekomendasi |
+------------------------------ | ------------ |
+Otorisasi RBAC (Akses terhadap API Kubernetes) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/ |
+Autentikasi | https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/ |
+Manajemen Secret Aplikasi (dan mengenkripsi mereka di etcd saat diam) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ |
+Pod Security Policy | https://kubernetes.io/docs/concepts/policy/pod-security-policy/ |
+Quality of Service (dan manajemen sumber daya kluster) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/ |
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Quality of Service (dan manajemen sumber daya klaster) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/ |
```

File: content/id/docs/concepts/security/overview.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+
+### Komponen-komponen _di dalam_ Cluster (aplikasimu)
+
+Bergantung pada permukaan yang dapat diserang dari aplikasimu, kamu mungkin mau berfokus pada aspek keamanan yang spesifik. Sebagai contoh, jika kamu menjalankan sebuah layanan (kita sebut Layanan A) yang kritikal di dalam rantai sumber daya lainnya dan sebuah beban kerja terpisah (kita sebut Layanan B) yang rentan terhadap serangan _resource exhaustion_, dengan tidak menyetel limit untuk sumber daya maka kamu juga menaruh risiko terhadap Layanan A. Berikut tabel tautan-tautan menuju hal-hal yang perlu diperhatikan untuk mengamankan beban-beban kerja yang berjalan di dalam Kubernetes.
+
+Area yang Menjadi Perhatian untuk Keamanan Beban Kerja | Rekomendasi |
+------------------------------ | ------------ |
+Otorisasi RBAC (Akses terhadap API Kubernetes) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/ |
+Autentikasi | https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/ |
+Manajemen Secret Aplikasi (dan mengenkripsi mereka di etcd saat diam) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ |
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Manajemen Secret Aplikasi (dan mengenkripsi mereka di etcd) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/ |
```

File: content/id/docs/concepts/security/overview.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
...
+Pod Security Policy | https://kubernetes.io/docs/concepts/policy/pod-security-policy/ |
+Quality of Service (dan manajemen sumber daya kluster) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/ |
+Network Policy | https://kubernetes.io/docs/concepts/services-networking/network-policies/ |
+TLS untuk Ingress Kubernetes | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls |
+
+## Container
+
+Untuk menjalankan perangkat lunak di dalam Kubernetes, ia harus berada di dalam sebuah Container. Karenanya, ada beberapa pertimbangan keamanan yang harus diperhitungkan untuk mengambil manfaat dari fitur-fitur keamanan beban kerja Kubernetes. Keamanan Container berada di luar lingkup panduan ini, tetapi berikut disediakan sebuah tabel rekomendasi-rekomendasi umum dan tautan menuju eksplorasi lebih dalam pada topik ini.
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
Untuk menjalankan perangkat lunak di dalam Kubernetes, perangkat lunak tersebut haruslah berada di dalam sebuah Container. Karenanya, ada beberapa pertimbangan keamanan yang harus diperhitungkan untuk mengambil manfaat dari fitur-fitur keamanan beban kerja Kubernetes. Keamanan Container berada di luar lingkup panduan ini, tetapi berikut disediakan sebuah tabel rekomendasi-rekomendasi umum dan tautan menuju eksplorasi lebih dalam pada topik ini.
```

File: content/id/docs/concepts/security/overview.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+
+## Otomasi yang Kokoh
+
+Kebanyakan dari saran yang disebut di atas dapat diotomasi di dalam _delivery pipeline_ kode kamu sebagai bagian dari rangkaian pemeriksaan keamanan. Untuk mempelajari lebih lanjut tentang pendekatan "_Continuous Hacking_" terhadap _delivery_ perangkat lunak, [artikel ini](https://thenewstack.io/beyond-ci-cd-how-continuous-hacking-of-docker-containers-and-pipeline-driven-security-keeps-ygrene-secure/) menyediakan lebih banyak detail.
+
+{{% /capture %}}
+{{% capture whatsnext %}}
+
+* Pelajari tentang [Network Policy untuk Pod](/docs/concepts/services-networking/network-policies/)
+* Pelajari tentang [mengamankan kluster kamu](/docs/tasks/administer-cluster/securing-a-cluster/)
```
Comment (irvifa) [inline content/id/docs/concepts/security/overview.md]:
```suggestion
* Pelajari tentang [mengamankan klaster kamu](/docs/tasks/administer-cluster/securing-a-cluster/)
```

**Comment by phanama** [comment]
Pushed the updates! @irvifa

**Comment by irvifa** [comment]
Thank you 🎉 
/lgtm

**Pull request merged**